pages/login.vue

```diff
@@ -1,232 +1,102 @@
-        font-weight: bold;
-        unicode-bidi: isolate;
-    }
-
-    .text-input-wrapper:focus-within {
-        border-color: rgb(11, 125, 224) !important;
-    }
-
-    .text-input-wrapper:not(:focus-within) .cancel-email-input-button {
-        display: none;
-    }
-
-    .text-input-wrapper {
-        background: rgb(var(--color-polar));
-        /* border: 2px solid rgb(var(--input-primary-color)); */
-        border: 2px solid rgb(31, 36, 40);
-        border-radius: 12px;
-        display: flex;
-        overflow: hidden;
-        margin-top: 8px;
-        width: 100%;
-    }
-
-    .cancel-email-input {
-        height: 24px;
-        width: 24px;
-        user-select: inherit;
-        background: none;
-
-    }
+            <h3 class="text-eel mx-auto mb-8">Log in</h3>
 
-    .cancel-email-input-button {
-        align-self: center;
-        margin-left: 2px;
-        margin-right: 6px;
-        border: none;
-        padding: 0;
-        transition: filter .2s;
-        -webkit-tap-highlight-color: transparent;
-        -webkit-touch-callout: none;
-        cursor: pointer;
-        display: inline-flex;
-        outline: none;
-        position: relative;
-        touch-action: manipulation;
-        transform: translateZ(0);
-    }
-
-    * :after, :before, button, button:focus, input, textarea{
-        -webkit-tap-highlight-color: transparent;
-        box-sizing: inherit;
-        font-family: din-round, sans-serif;
-        outline: none;
-        -webkit-user-select: none;
-        -moz-user-select: none;
-        user-select: none;
-    }
-    img {
-        overflow-clip-margin: content-box;
-        overflow: clip;
-    }
-
-    .forgot-button {
-        -webkit-font-smoothing: antialiased;
-        align-self: center;
-        color: rgb(var(--color-hare));
-        font: 700 15px / 28px din-round, sans-serif;
-        letter-spacing: .8px;
-        margin-left: 2px;
-        margin-right: 14px;
-        text-decoration: none;
-        text-transform: uppercase;
-    }
-
-    .alt-login-divider {
-        align-items: center;
-        display: flex;
-        justify-content: center;
-        margin: 20px 0;
-        width: 100%;
-    }
-
-    .horizontal-bar {
-        background: rgb(0,0,0);
-        flex-grow: 1;
-        height: 2px;
-        display: block;
-        unicode-bidi: isolate;
-    }
-
-    .or-styling {
-        color: rgb(var(--color-hare));
-        flex-grow: 0;
-        padding: 0 8px;
-    }
+            <LAInput
+              v-model="email"
+              placeholder="Email"
+              type="text"
+              class="mb-4"
+            >
+              <button
+                @click="email = ''"
+                class="d-inline-flex align-self-center highlight"
+                type="button"
+              >
+                <img src="../assets/cancel-circle.svg" /></button
+            ></LAInput>
+
+            <LAInput v-model="password" placeholder="Password" type="password">
```

Comment:
Both LAInput and LAButton have slots now for extra stuff that we want to append. Shouldn't need to worry about managing wrappers for these anymore.